src/qudi/hardware/interfuse/ni_scanning_probe_interfuse.py

```diff
@@ -429,8 +430,10 @@ def stop_scan(self):
         """
         #self.log.debug("Stopping scan")
         if self.thread() is not QtCore.QThread.currentThread():
-            QtCore.QMetaObject.invokeMethod(self, '_stop_scan',
-                                            QtCore.Qt.BlockingQueuedConnection)
+            #QtCore.QMetaObject.invokeMethod(self, '_stop_scan',
+            #                                QtCore.Qt.BlockingQueuedConnection)
+            # FIXME: why does the above not work anymore?
+            self._stop_scan()
```

Comment:
Not sure why, but somehow `invokeMethod` does not work anymore if the tilt correction mixin is used. It complains that there is no such method on `NIScanningProbeInterfuse`. It works if only the bare interfuse is loaded.

Reply:
`invokeMethod` works in mysterious ways in `PySide2`. One of those mysteries is, that it only works if the slot in question has been decorated with the `PySide2.QtCore.Slot` decorator.

It's one of many reasons why using signals is preferred over `invokeMethod`.

Reply:
I was able to resolve this issue by re-implementing `_start_scan` and `_stop_scan` in the class which inherits the bare scanner and the tilt correction mixin.